internal/pkg/manifest/workload_ecs.go

```diff
@@ -85,13 +85,57 @@ type ImageWithPortAndHealthcheck struct {
 	HealthCheck   ContainerHealthCheck `yaml:"healthcheck"`
 }
 
+// AlarmArgs represents specs of CloudWatch alarms for deployment rollbacks.
+type AlarmArgs struct {
+	CPUUtilization    *int `yaml:"cpu_utilization"`
+	MemoryUtilization *int `yaml:"memory_utilization"`
+}
+
+// RollbackAlarmArgsOrNames represents alarms for rolling back deployments.
+// It supports unmarshalling yaml which can either be of type AlarmArgs or a list of strings.
+type RollbackAlarmArgsOrNames struct {
+	AlarmNames []*string
```

Comment:
Why is this a list of pointer to string? Is this because of `mergo`?

Reply:
changed back 👍🏼